tools/dependencies/known-oap-backend-dependencies.txt

```diff
@@ -117,26 +120,23 @@ lucene-spatial-extras-7.3.1.jar
 lucene-spatial3d-7.3.1.jar
 lucene-suggest-7.3.1.jar
 lz4-java-1.6.0.jar
-minimal-json-0.9.5.jar
 moshi-1.5.0.jar
 msgpack-core-0.8.16.jar
 mvel2-2.4.8.Final.jar
 nacos-api-1.4.2.jar
 nacos-client-1.4.2.jar
 nacos-common-1.4.2.jar
-netty-buffer-4.1.65.Final.jar
-netty-codec-4.1.65.Final.jar
-netty-codec-dns-4.1.65.Final.jar
+netty-buffer-4.1.51.Final.jar
+netty-codec-4.1.51.Final.jar
 netty-codec-http-4.1.65.Final.jar
 netty-codec-http2-4.1.51.Final.jar
 netty-codec-socks-4.1.51.Final.jar
-netty-common-4.1.65.Final.jar
+netty-common-4.1.51.Final.jar
 netty-handler-4.1.65.Final.jar
 netty-handler-proxy-4.1.51.Final.jar
-netty-resolver-4.1.65.Final.jar
-netty-resolver-dns-4.1.65.Final.jar
+netty-resolver-4.1.51.Final.jar
 netty-tcnative-boringssl-static-2.0.39.Final.jar
-netty-transport-4.1.65.Final.jar
+netty-transport-4.1.51.Final.jar
```

Comment:
I don't think you should change these versions, they are upgraded before due to CVEs